routers/api/v1/repo/issue_tracked_time.go

```diff
@@ -117,7 +121,7 @@ func AddTime(ctx *context.APIContext, form api.AddTimeOption) {
 
 	if !ctx.Repo.CanUseTimetracker(issue, ctx.User) {
 		if !ctx.Repo.Repository.IsTimetrackerEnabled() {
-			ctx.JSON(400, struct{ Message string }{Message: "time tracking disabled"})
+			ctx.Error(400, "", "time tracking disabled")
```

Comment:
This change the response format breaking backward compat.But since the json format wasn't adding anything and should be more inline we the existing swagger definition I don't think this is blocking.

Reply:
@6543 yes, there is just very few response like that. I am ok with this I just wanted to point that we maybe need a generic JSONError response for another PR and that it may had impact if someone use api (even if it should be very limited). The only "blocking" point would be the http.Status for me.